bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/exceptions/AssetNotLoadedException.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hue.internal.exceptions;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Thrown when one of the connection or handler classes has not loaded all its assets.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@SuppressWarnings("serial")
```

Comment:
Why is this suppression needed?

Reply:
Because Maven the compiler asks for it. :)

Reply:
It gave you a detailed warning, but didn't ask you to suppress it. 🙂 "The serializable class AssetNotLoadedException does not declare a static final serialVersionUID field of type long". So I would propose to just add:

```java
private static final long serialVersionUID = -1;
```

Reply:
Ok.